backend/infrahub/workflows/utils.py

```diff
@@ -79,8 +80,7 @@ async def wait_for_schema_to_converge(
             log.warning(
                 f"Schema had not converged after {delay * iteration:.2f} seconds, refreshing schema on local worker manually"
             )
-            async with service.database.start_session() as db:
```

Comment:
I think we do not need to create a session for each `refresh_branches` if we created one earlier